### ISSUE TITLE
isList, isMap, isOrderedList, isOrderedMap e.t.c. are not exported

### ISSUE THREAD

**Issue by GolubevS** [immutable-js/immutable-js#1740]
### What happened

isList, isMap, isOrderedList, isOrderedMap e.t.c. are not exported from immutable.es.js

### How to reproduce
```
import {Map, isOrderedMap} from 'immutable';
console.log(isOrderedMap(Map({b:1,a:2}).sort()));
```

Result: 

export 'isOrderedMap' was not found in 'immutable'

**Comment by toshipon** [comment]
it should be like this ? 
```
import {Map, OrderedMap} from 'immutable';
console.log(OrderedMap.isOrderedMap(Map({b:1,a:2}).sort()));
```

**Comment by jdeniau** [comment]
Thank you for your bug report. The immutable-js oss fork will soon be merged and a fix for this issue has been included in #1833. Once this PR is merged, this issue will be resolved in the main branch. We will then do our best to to release the `4.0.0` version.

Commit reference: https://github.com/immutable-js/immutable-js/pull/1833/commits/b86aebd8f9756f31cff3e2d8c92759f633b5a7f0
